app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -217,7 +217,7 @@ class AdvancedFilter extends React.Component {
     if (savedFilters.length > 0) {
       _.forEach(savedFilters, (filter, filterIndex) => {
         _.forEach(filter.contents, (statement, statementIndex) => {
-          if (statement.filterOption.type == 'multi') {
+          if (statement.filterOption?.type == 'multi') {
```

Comment:
out of curiosity, what case warranted this to be added? (for testing purposes)

Reply:
I would run into a front end error after creating these new filters and checking out an old branch

Reply:
Each filter option should have a type so I'm not really sure why this would cause an error (I was also unable to recreate).  The extra check isn't a big deal, just trying to understand

Reply:
I think it was that the `filtlerOption` was null/undefined and then I was getting the error saying "can't get `type` of undefined"

Reply:
hmm that also shouldn't happen the way filters are currently saved, but there's a ticket out there to refactor how the filters are saved anyway so I'm not worried about it